src/main/io/osd.c

```diff
@@ -927,14 +927,15 @@ static bool osdDrawSingleElement(uint8_t item)
             tfp_sprintf(buff, "%c%03d", osdGetDirectionSymbolFromHeading(heading), heading);
             break;
         }
-
+#ifdef USE_VARIO
```

Comment:
Your use of `#ifdef` and `#if defined()` is very inconsistent.

Reply:
All uses of defined() have been removed.